get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+
+After [installing DVC][installdvc] and cloning the repository, you can run:
+
+[installdvc]: https://dvc.org/doc/install
+
+```console
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+```
+
+`params.yaml` defines many parameters to modify with `dvc exp run --set-param/-S`
+option. For example to set the number of convolutional units in `cnn` model, you
+can use:
+
+```console
+dvc exp run -S model.cnn.conv_units=128 
+```
+
+The experiment will produce a new `metrics.json` and you can check the changes
+in metrics:
+
+```console
+dvc exp diff
+```
+
+You can queue many experiments with `--queue` option:
+
+```console
+dvc exp run --queue -S model.cnn.dropout=0.1
+dvc exp run --queue -S model.cnn.dropout=0.2
+dvc exp run --queue -S model.cnn.dropout=0.3
+dvc exp run --queue -S model.cnn.dropout=0.4
+dvc exp run --queue -S model.cnn.dropout=0.5
+dvc exp run --queue -S model.cnn.dropout=0.6
+dvc exp run --queue -S model.cnn.dropout=0.7
+dvc exp run --queue -S model.cnn.dropout=0.8
+dvc exp run --queue -S model.cnn.dropout=0.9
+```
+
+and run them at once, preferably in parallel with `--jobs`:
+
+```console
+dvc exp run --run-all --jobs 4
+```
+
+You can get the summary of experiments with: 
+
+```console
+dvc exp show
+```
+
+and limit the parameters and metrics to show with `--include-params` and
+`--include-metrics` options, respectively.  
+
+Experiments are given hash-names derived from their inputs and environment. It
+may be easier to review them when you give names with `--name/-n` option.
+
+```console
+dvc exp run -n baseline-experiment
+```
+
+Experiments are normally not checked-in to Git. When there is an experiment that
+you want to preserve in Git history, you can use:
+
+```console
+dvc exp apply exp-123456
+```
+
+Then you can use `git add`, `git commit`, `git push` and `dvc push` as usual. 
+
+You can push and pull the _code changes_ related to an experiment with `dvc exp
+push` and `dvc exp pull` respectively. These two commands work with _Git
+remotes._
+
+You can clean up the experiments that don't make into the repository with:
+
+```console
+dvc exp gc --workspace
+```
+
+### Pipeline and Parameters
+
+The pipeline defined in `dvc.yaml` consists of four stages that depend on each
+other sequentially: `prepare`, `preprocess`, `train` and `evaluate`. `train`
+stage also depends on `src/models.py` that defines two simple models, `mlp` and
+`cnn`.  
+
+When you change a parameter for an earlier stage, later stages depend on this
+stage are also run by `dvc exp run`. So when you change a parameter for
+`prepare`, all the later stages are run. 
+
+Notable parameters for each stage that you can use with `--set-param` option are
+as follows: 
+
+- `prepare.remix`: Determines whether Fashion-MNIST train (60000 images) and
+  test (10000 images) sets are merged and split. If `false`, the test and
+  train sets are not merged and used as in the original.
+
```

Comment:
inter-bullet spacing is annoying in this long list - please remove

Reply:
OK.I would like to use definition lists but this is not RST.

Reply:
I agree since most elements are short sentences the bullet list doesn't need spacing but "is annoying" wasn't a good reason haha. Annoying to whom? Users see rendered markdown 🙂